scala-package/examples/scripts/infer/objectdetector/run_ssd_example.sh

```diff
@@ -17,9 +17,21 @@
 # specific language governing permissions and limitations
 # under the License.
 
+hw_type=cpu
+if [[ $1 = gpu ]]
+then
+    hw_type=gpu
+fi
+
+platform=linux-x86_64
+
+if [[ $OSTYPE = [darwin]* ]]
```

Comment:
where are you getting this variables from `$OSTYPE` and `$1` is set to MODEL_DIR

Reply:
Fixing the $1 thing. I missed this when I took the suggestion to update the script to not be osx specific.

The $OSTYPE is an env variable.

Reply:
$1 thing is to pick up CPU or GPU because you need to append that as well to the path.

Reply:
$1 was to MODEL_DIR